imperative/python/megengine/functional/tensor.py

```diff
@@ -104,38 +104,35 @@ def full(
     dtype=None,
     device=None,
 ) -> Tensor:
-    r"""Creates a tensor of shape ``shape`` filled with ``value``.
+    r"""Return a tensor of shape ``shape`` filled with ``value``.
 
     Args:
         shape: output tensor shape.
         value: fill value.
-        dtype: output tensor data type. If ``dtype`` is ``None``, the output tensor
+        
+    Keyword args:
+        dtype(:attr:`.Tensor.dtype`): output tensor data type. If ``dtype`` is ``None``, the output tensor
             data type must be inferred from ``value``. If the value is an ``int``,
             the output tensor data type must be the default integer data type. If the
             value is a ``float``, the output tensor data type must be the default
             floating-point data type. If the value is a ``bool``, the output tensor 
-            must have boolean data type. Default: ``None``.
-        device: device on which to place the created tensor. Default: ``None``.
+            must be a boolean data type. Default: ``None``.
+        device(:attr:`.Tensor.device`): device on which to place the created tensor. Default: ``None``.
```

Comment:
```suggestion
        device (:attr:`.Tensor.device`): device on which to place the created tensor. Default: ``None``.
```

出于排版阅读性要求，这里统一加一个空格。